macros/charmap.asm

```diff
@@ -152,7 +188,7 @@
 	charmap "♂",        $ef
 	charmap "¥",        $f0
 	charmap "×",        $f1
-	charmap "·",        $f2
+	charmap "<PERIOD>", $f2 ; same as "." in English
```

Comment:
`·` is a different character (https://en.wikipedia.org/wiki/Interpunct). `<DOT>` might be better.

Reply:
The tile looks identical to the period; it's used as a decimal point in PrintNum, and sometimes to abbreviate "`<ID>№`" (other times a period or slash is used). I'll rename it to "`<DOT>`".

Reply:
hmm, which one am i thinking of then. there's one that is a dot in the middle. or was that the ellipsis.

Reply:
You're thinking of `charmap "·", $74 ; middle dot, unused` in gfx/misc/font_extra.png.